index.html

```diff
@@ -5,9 +5,374 @@
+                <li><a href="#">U.S.</a></li>
+                <li><a href="#">Politics</a></li>
+            </ul>
+        </nav>
+        <nav class="nav-column column-list">
+            <h3 class="link-list-header">opinion</h3>
+            <ul>
+                <li><a href="#">Today's</a></li>
+                <li><a href="#">Op-Ed</a></li>
+                <li><a href="#">Editorials</a></li>
+                <li><a href="#">Letters</a></li>
+            </ul>
+        </nav>
+        <nav class="nav-column column-list">
+            <h3 class="link-list-header">arts</h3>
+            <ul>
+                <li><a href="#">Today's Arts</a></li>
+                <li><a href="#">Art & Design</a></li>
+                <li><a href="#">Books</a></li>
+                <li><a href="#">Dance</a></li>
+            </ul>
+        </nav>
+        <nav class="nav-column column-list">
+            <h3 class="link-list-header">living</h3>
+            <ul>
+                <li><a href="#">Automobiles</a></li>
+                <li><a href="#">Crossword</a></li>
+                <li><a href="#">Education</a></li>
+                <li><a href="#">Food</a></li>
+            </ul>
+        </nav>
+        <nav class="nav-column column-list">
+            <h3 class="link-list-header">more</h3>
+            <ul>
+                <li><a href="#">Reader</a></li>
+                <li><a href="#">Wirecutter</li>
+                <li><a href="#">Live Events</a></li>
+                <li><a href="#">The Learning Network</a></li>
+            </ul>
+        </nav>
+        <nav class="nav-column column-subscribe">
+            <h3 class="link-list-header">subscribe</h3>
+            <ul>
+                <li><a href="#">Home Delivery</a></li>
+                <li><a href="#">Digital Subscription</a></li>
+                <li><a href="#">Crossword</li>
+                <li><a href="#">Cooking</a></li>
+            </ul>
+        </nav>
+    </div>
+    <footer class="nav-footer">
+        <nav>
+            <a href="#" class="nav-footer-links">© 2019 The New York Times Company</a>
+            <a href="#" class="nav-footer-links">Contact Us</a>
+            <a href="#" class="nav-footer-links">Work with us</a>
+            <a href="#" class="nav-footer-links">Advertise</a>
+            <a href="#" class="nav-footer-links">Your Ad Choices</a>
+            <a href="#" class="nav-footer-links">Privacy</a>
+            <a href="#" class="nav-footer-links">Terms of Service</a>
+            <a href="#" class="nav-footer-links">Terms of Sale</a>
+            <a href="#" class="nav-footer-links">Site Map</a>
+            <a href="#" class="nav-footer-links">Help</a>
+            <a href="#" class="nav-footer-links">Subscriptions</a> 
+        </nav>
+    </footer>
+    <div class="bottom-spacer"></div>
+    <div class="bottom-ad">
+        <p>Just $1 a week, plus Cooking and Crosword free. Today only.</p>
+        <img src="./resources/bottom-add-icon.png" alt="expand icon">
```

Comment:
You cannot use an image here. You need to create these link icons. Use https://origin.fontawesome.com/start. They don't need to be exactly the same.